proposal.md

```diff
@@ -44,6 +58,27 @@ the project requires a contributor license agreement or other
 procedural steps, please explain here.  "None at this time" is
 acceptable for this question.
 
+* The Clawpack community was most active approximately 5-10 years ago and is less active these days. It may not be the best choice going forward for a project.
+
+    * PyClaw has the most contributors and lots of documentaiton (would be the best solver of the four to choose for a project).
+    * GeoClaw had the most changes in the past year, but only one contributor.
+    * Submitted request to join Clawpack Slack channel but have yet to hear back from them.
+    
+* Travis continuos integration is used (service used to build and test software projects hosted on GitHub and Bitbucket).
+
+* If using either the Fortran or Python installations there is an automated test suite that runs several tests and compares the solution (parts of the soltution) to archived resutls.
+
+    * For PyClaw, 46 of 48 tests in the test suite pass and two tests fail. The two tests failed with the error "no such file or directory" for the verification file (the file name for the verificaiton file might be mislabeld for those tests).
+    * For the calssic solver, all three tests in the test suite failed.
```

Comment:
Hmm, it looks like continuous integration isn't running any more and I see an issue about Travis (which doesn't provide free minutes to open source any more).
https://app.travis-ci.com/github/clawpack/clawpack/builds